gitpod-io/website
# Add instructions for collecting logs in JetBrains IDEs

## Description

Add instructions for collecting logs in JetBrains IDEs.

It's ok to have the paragraphs duplicated between the sections on this page, as the user might land to this Troubleshooting page via a link with anchor to specific section [[1](https://github.com/gitpod-io/website/pull/1959#discussion_r859870788)].

## Related Issue(s)

Resolves #1636

## How to test

- Visit the [Troubleshooting page in this Preview Environment](https://deploy-preview-1959--gitpod-kumquat.netlify.app/docs/troubleshooting#gitpod-logs-in-jetbrains-ides)
- Check if the content from "Gitpod logs in JetBrains IDEs" section is fine.

## Release Notes

```release-note
Instructions for collecting logs in JetBrains IDEs were added on the Troubleshooting page.
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1959"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1959"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
